src/main/java/edu/greenblitz/robotName/subsystems/shooter/Pivot/FalconPivot/FalconPivot.java

```diff
@@ -1,27 +1,53 @@
 package edu.greenblitz.robotName.subsystems.shooter.Pivot.FalconPivot;
 
-import com.ctre.phoenix6.controls.MotionMagicDutyCycle;
+import com.ctre.phoenix6.BaseStatusSignal;
 import com.ctre.phoenix6.controls.PositionVoltage;
 import com.ctre.phoenix6.hardware.TalonFX;
 import com.ctre.phoenix6.signals.NeutralModeValue;
+import edu.greenblitz.robotName.RobotConstants;
 import edu.greenblitz.robotName.subsystems.shooter.Pivot.IPivot;
-import edu.greenblitz.robotName.subsystems.shooter.Pivot.PivotConstants;
 import edu.greenblitz.robotName.subsystems.shooter.Pivot.PivotInputsAutoLogged;
 import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
 
+import static edu.greenblitz.robotName.RobotConstants.General.Motors.IS_SWITCH_CLOSED;
 import static edu.greenblitz.robotName.subsystems.shooter.Pivot.FalconPivot.FalconPivotConstants.*;
 
 public class FalconPivot implements IPivot {
 
     private TalonFX motor;
+    
+    private DutyCycleEncoder absoluteEncoder;
 
+    private PositionVoltage pidController;
```

Comment:
rename to positionVoltage
```suggestion
    private PositionVoltage positionVoltage;
```